src/opengl/text.rs

```diff
@@ -281,13 +293,13 @@ impl TextRenderer {
         &mut self,
         surface: &mut S,
         text: &str,
-        x: f32,
-        y: f32,
-        color: [f32; 3],
+        pos: [f32; 2],
```

Comment:
Why `[f32; 2]` versus `(f32, f32)`? just curious, seen this a bunch throughout the code.

Reply:
Not much reason behind this, if it's better the other way that's fine as well

Reply:
oh i don't think it's beter, just wondering